smartcar-auth/src/main/java/com/smartcar/sdk/SmartcarAuth.java

```diff
@@ -123,13 +123,18 @@ public AuthUrlBuilder setApprovalPrompt(boolean approvalPrompt) {
             return this;
         }
 
+        public AuthUrlBuilder setMakeBypass(String make) {
+            urlBuilder.addQueryParameter("make", make);
+            return this;
+        }
+
         public AuthUrlBuilder setSingleSelect(boolean singleSelect) {
             urlBuilder.addQueryParameter("single_select", Boolean.toString(singleSelect));
             return this;
         }
 
-        public AuthUrlBuilder setMakeBypass(String make) {
-            urlBuilder.addQueryParameter("make", make);
+        public AuthUrlBuilder setSingleSelectVin(String vin) {
+            urlBuilder.addQueryParameter("single_select_vin", vin);
```

Comment:
With the Java/Android SDK, do we want to force the developer to explicitly specify `.setSingleSelect(true)` in addition to `.setSingleSelectVin(...)`?

```java
.setSingleSelect(true)
.setSingleSelectVin(true)
```
vs.
```java
.setSingleSelectVin(true)

Reply:
yes we explicitly require that they set both!